src/pages/myNotes/note/note.jsx

```diff
@@ -1,24 +1,31 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Box } from '@material-ui/core';
-import { cutSymbols } from '../../../utils';
-import useStyles from '../styled';
 
-const Note = ({ activeNote: { title, description, date, id }, notActive, changeNoteContent }) => {
-  const styles = useStyles();
-  return (
-    <Box
-      className={notActive === id ? styles.activeNote : styles.note}
-      onClick={() => {
+import { cutSymbols } from 'utils';
+
+import { Title, Date, DefaultNote, ActiveNote } from './styled';
+
+const Note = ({ activeNote: { title, description, date, id }, notActive, changeNoteContent }) => (
+  <>
+    {notActive === id
```

Comment:
Can you please rename all the booleans starting with: `isActive` or `isNotActive` (the first one is better to use, don't forget to rewrite logic)